src/org/daisy/dotify/translator/impl/sv_SE/SwedishContractedBrailleFilter.java

```diff
@@ -0,0 +1,89 @@
+package org.daisy.dotify.translator.impl.sv_SE;
+
+import org.daisy.dotify.common.text.StringFilter;
+
+import java.io.IOException;
+import java.net.URL;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Properties;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ *
+ */
+public class SwedishContractedBrailleFilter implements StringFilter {
+
+    private HashMap<String, String> contractedBrailleMap;
+    private static final String CONTRACTED_BRAILLE_TABLE_PATH = "sv_SE-single-character-contracted-words.xml";
+
+    public static final String CAPITAL_CHAR_MARKER = "\\u2820";
+    public static final String SOFT_HYPHEN = "\\u2820";
```

Comment:
Capital char marker and a soft hyphen is not the same char. Soft hyphen = \\u00AD

Reply:
Oops, to much copy past. i Will fix this.